addon/components/frost-popover.js

```diff
@@ -164,11 +180,7 @@ export default Component.extend(PropTypeMixin, {
    * @param {DOMEvent} event - click event
    */
   togglePopover (event) {
-    const popover = this.get('element')
-    if ($(event.target).closest(popover).length === 0 ||
```

Comment:
So does this remove the ability to have a sustained popup and clicking on the outside to close it?

Reply:
Crap - yes.

Reply:
Actually, no - this still works because I am blocking the click on the content from propagating. But, what does happen is that if the popover is meant to work on click and you hover over it - it will now close on mouseleave.